src/main/java/com/mitrais/chipper/temankondangan/backendapps/service/impl/EventServiceImpl.java

```diff
@@ -87,7 +80,8 @@ public Event create(Long userId, CreateEventWrapper wrapper) {
 		} else if (direction.equalsIgnoreCase("ASC")) {
 			paging = PageRequest.of(pageNumber, pageSize, Sort.by(sortBy).ascending());
 		} else {
-			throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "Direction error!");
+			throw new ResponseStatusException(HttpStatus.BAD_REQUEST,
```

Comment:
I think it should throws new BadRequestException

Reply:
oh yeah forgot haha

Reply:
wait, it is in the outdated version haha, I've changed it